glusterd2/bricksplanner/planner.go

```diff
@@ -141,17 +141,25 @@ func getBricksLayout(req *api.VolCreateReq) ([]api.SubvolReq, error) {
 			}
 			eachBrickTpSize := uint64(float64(eachBrickSize) * req.SnapshotReserveFactor)
 
+			mntopts := "rw,inode64,noatime,nouuid,discard"
+			if req.ProvisionerType == api.ProvisionerTypeLoop {
+				mntopts += ",loop"
+			}
+
+			tpsize := lvmutils.NormalizeSize(eachBrickTpSize)
+			tpmsize := lvmutils.GetPoolMetadataSize(eachBrickTpSize)
```

Comment:
tpsize and tpmsize also needs to be different per ProvisionerType ?

Reply:
I will change that. Both the functions are not related to LVM(don't use lvm commands), so I didn't change those in first iteration.